pykickstart/commands/timezone.py

```diff
@@ -116,18 +117,24 @@ def __init__(self, writePriority=0, *args, **kwargs):
 
     def __str__(self):
```

Comment:
Instead of populating this with isinstance checks let's just make a new one for the F32 class. Keeps the logic cleaner and makes it easier to read.